glide.yaml

```diff
@@ -18,10 +19,10 @@ import:
 
 # crd-schema-gen
 - package: sigs.k8s.io/controller-tools
-  version: 43466124052c1a1aa7d8fd33624b00bc111fb7cf
-  repo: https://github.com/openshift/kubernetes-sigs-controller-tools.git
+  repo: https://github.com/openshift/kubernetes-sigs-controller-tools
+  version: master
 - package: k8s.io/gengo
-  version: 4242d8e6c5dba56827bb7bcf14ad11cda38f3991
+  version: master
```

Comment:
this is wrong.  Choose a specific level.

Reply:
added specific commit.